src/DevBetterWeb.Core/Events/DailyCheckEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using DevBetterWeb.Core.SharedKernel;
+
+namespace DevBetterWeb.Core.Events
+{
+  public class DailyCheckEvent : BaseDomainEvent
```

Comment:
Event names should be past tense. What happened? DailyCheckCompleted?

Reply:
well... it's not that the daily check was completed, it's an event saying "hey do the daily check." I'm not sure what a better name would be in that sense. Do you have any ideas?

Reply:
"Initiated" "Started"